tests/xtro-sharpie/u2ignore/u2ignore.csproj

```diff
@@ -1,40 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{8038D233-1BAB-4228-A694-EA190928CA4A}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <RootNamespace>u2ignore</RootNamespace>
-    <AssemblyName>u2ignore</AssemblyName>
-    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
-    <LangVersion>latest</LangVersion>
+    <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
Any reason to not have `<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>` as well for consistency?

Reply:
Fixed.